lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -2525,6 +2525,88 @@ Value *PatchInOutImportExport::patchFsBuiltInInputImport(Type *inputTy, unsigned
     input = getSamplePosition(inputTy, insertPos);
     break;
   }
+  case BuiltInBaryCoord: {
+    assert(entryArgIdxs.perspInterp.center != 0);
+    builder.SetInsertPoint(insertPos);
+    auto iJCoord = getFunctionArgument(m_entryPoint, entryArgIdxs.perspInterp.center);
+    auto iCoord = builder.CreateExtractElement(iJCoord, uint64_t(0));
+    auto jCoord = builder.CreateExtractElement(iJCoord, 1);
+    unsigned primitiveMode = m_pipelineState->getVerticesPerPrimitive();
+    switch (primitiveMode) {
+    case 1: {
+      // points
+      input = builder.CreateInsertElement(input, ConstantFP::get(Type::getFloatTy(*m_context), 1.0), uint64_t(0));
+      input = builder.CreateInsertElement(input, ConstantFP::get(Type::getFloatTy(*m_context), 0.0), 1);
+      input = builder.CreateInsertElement(input, ConstantFP::get(Type::getFloatTy(*m_context), 0.0), 2);
+      break;
+    }
+    case 2: {
+      // lines
+      input = builder.CreateInsertElement(input, iCoord, uint64_t(0));
```

Comment:
It is better to add not_tested or not_implemented here.

Reply:
Okay, I will fix it.